### PR TITLE
CONTRIBUTING.md: Initial release

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
As per issue #2623, it is important to define the commit guidelines.
This commit is about adding a first version of it.

### CODE REVIEW

**Comment by guits** [comment]
@ErwanAliasr1 Could we suggest to add the `Fixes: #XXXX` if an issue is linked to the PR?

**Comment by ErwanAliasr1** [comment]
@guits done.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+CI
+-----
+
+### Jenkins
+We use Jenkins to run several tests on each pull request.
+
+If you don't want to run a build for a particular pull request, because all you are changing is the
+README for example, add the text `[skip ci]` to the PR title.
+
+### Travis
```
Comment (leseb) [inline CONTRIBUTING.md]: We don't use Travis on ceph-ansible. Please remove the section.

**Comment by ErwanAliasr1** [comment]
Oups. Fixed. thx.

**Comment by guits** [comment]
@andrewschoen @alfredodeza @leseb

I'm merging this as it is. Please, don't hesitate to update/propose/suggest if you have any concerns/ideas

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Contributing to ceph-ansible
+==============================
+
+1. Follow the [commit guidelines](#commit-guidelines)
+
+
+Commit guidelines
+-----------------
+- All commits should have a subject and a body
```
Comment (alfredodeza) [inline CONTRIBUTING.md]: Is this going to be forcefully checked regardless? Sounds a bit extreme to require a body for every single commit

Comment (ErwanAliasr1) [reply CONTRIBUTING.md]: This is not yet forcefully check but will surely.
That is not extreme when considering supporting the software.
When you have to debug, a commit message is always useful to understand why that change was made. A single line patch can lead to a very long commit message.

So yes, a detailed commit message is mandatory.

**Pull request merged**